File: ClientSide/src/app/pages/faculties/models/studyProgrammeModel.ts
export class StudyProgrammeModel {

    public id: number;
    public code: string;
    
    public titleEn: string;
    public titleLv: string;
    public titleRu: string;

    public studentsCount: number;

    constructor(
        id: number,
        code: string,
        titleEn: string,
        titleLv: string,
        titleRu: string,
        studentsCount: number
    ) {
        this.id = id;
        this.code = code;
        this.titleEn = titleEn;
        this.titleLv = titleLv;
        this.titleRu = titleRu;
        this.studentsCount = studentsCount;
    }
}

File: ServerSide/SAMS.BusinessLogic/Models/StudyProgrammes/StudyProgrammeModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SAMS.BusinessLogic.Models.StudyProgrammes
{
    public class StudyProgrammeModel
    {
        public int Id { get; set; }
        public string Code { get; set; }

        public string TitleLv { get; set; }
        public string TitleRu { get; set; }
        public string TitleEn { get; set; }

        public int StudentsCount { get; set; }
    }
}
